7.Multiparty-UICollectionView/7.Multiparty-UICollectionView.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,389 @@
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
+				ENABLE_NS_ASSERTIONS = NO;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				GCC_C_LANGUAGE_STANDARD = gnu99;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				MTL_ENABLE_DEBUG_INFO = NO;
+				SDKROOT = iphoneos;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				VALIDATE_PRODUCT = YES;
+			};
+			name = Release;
+		};
+		F852CCD21EA4D88200ADB206 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = EC7306D4D7B7F6C8879263DC /* Pods-7.Multiparty-UICollectionView.debug.xcconfig */;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				INFOPLIST_FILE = "7.Multiparty-UICollectionView/Info.plist";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.tokbox.--Multiparty-UICollectionView";
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 3.0;
+			};
+			name = Debug;
+		};
+		F852CCD31EA4D88200ADB206 /* Release */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 313E7512ACAB0956CE7EC5BE /* Pods-7.Multiparty-UICollectionView.release.xcconfig */;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				INFOPLIST_FILE = "7.Multiparty-UICollectionView/Info.plist";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.tokbox.--Multiparty-UICollectionView";
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
+				SWIFT_VERSION = 3.0;
+			};
+			name = Release;
+		};
+/* End XCBuildConfiguration section */
+
+/* Begin XCConfigurationList section */
+		F852CCBA1EA4D88200ADB206 /* Build configuration list for PBXProject "7.Multiparty-UICollectionView" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				F852CCCF1EA4D88200ADB206 /* Debug */,
+				F852CCD01EA4D88200ADB206 /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+		F852CCD11EA4D88200ADB206 /* Build configuration list for PBXNativeTarget "7.Multiparty-UICollectionView" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				F852CCD21EA4D88200ADB206 /* Debug */,
+				F852CCD31EA4D88200ADB206 /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+/* End XCConfigurationList section */
+	};
+	rootObject = F852CCB71EA4D88200ADB206 /* Project object */;
+}
```

Comment:
And we need to remove changes generated from CocoaPods from this file. You can do `pod deintegrate` 😃